capa/render/result_document.py

```diff
@@ -540,3 +580,43 @@ def from_capa(cls, meta, rules: RuleSet, capabilities: MatchResults) -> "ResultD
             )
 
         return ResultDocument(meta=Metadata.from_capa(meta), rules=rule_matches)
+
+    @classmethod
+    def parse_raw(cls, path: str):
+        with open(path, "rb") as f:
+            buf= f.read()
+        data = json.loads(buf)
+        return ResultDocument(**data)    
+        
+    def to_capa(self, rules: RuleSet) -> Tuple[Dict, Dict]:
```

Comment:
~would you add a comment here explaining why the `rules` parameter is required? on first glance, it might be surprising that this information is required to convert from the serialized document~